superset/assets/src/dashboard/actions/dashboardFilters.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/* eslint-disable camelcase */
```

Comment:
why do we need this? why not just call it `formData`?

Reply:
In a few places in our code base, `form_data` implies the query params stored in slices table `from backend`, while `formData` is the updated query params after user changed UI controls from explore view or applied a dashboard-level filters, and this formData will be the new query parameters `sent to backend`. for example: https://github.com/apache/incubator-superset/blob/df9efa8ed9f6ee263f52b791086b286761e9aab6/superset/assets/src/dashboard/reducers/getInitialState.js#L104

In this place even we don't need both formData and form_data props at the same time, I still want to follow this rule, `form_data` means original query params.